### ISSUE TITLE
500 Internal Server Error when trying to add a site to Keystatic Cloud project

### ISSUE THREAD

**Issue by yeskunall** [Thinkmill/keystatic#1159]
I tried saving my project on Keystatic Cloud after adding a site to the project URLs section, and I was hit with a 500 Internal Server Error. Upon refreshing, however, everything looks good, and the site was saved successfully.

I have been able to replicate this error across 2 projects on 2 separate teams. Screenshot below might help.

Reproduction steps:

1. Setup a new project on Keystatic Cloud
2. Try adding a site to the Project URLs section
3. Save settings

You should see the default Vercel 500 error page.

Thanks for looking into it.

---
<details><summary>Console error</summary>
<p>
<img width="507" alt="CleanShot 2024-05-26 at 17 04 51@2x" src="https://github.com/Thinkmill/keystatic/assets/14703164/c89253f0-2a0a-4adb-be3a-840c2add68ae">
</p>
</details>

**Comment by emmatown** [comment]
Thanks for the report! I wasn't able to reproduce this consistently but I've made some changes and now can't reproduce it all so could you see if it's fixed for you?

**Comment by emmatown** [comment]
Thanks for the report! I believe this is fixed now, I'm not 100% sure since I wasn't able to reproduce this consistently so could you confirm if it's fixed for you now?

**Comment by yeskunall** [comment]
Hey Emma! I currently cannot follow the same steps I outlined above*. But, I did try adding a project URL, removing the current one, saving, adding multiple ones—and I was unable to reproduce it this time around.

I’ll leave it up to you to close the issue (or not). Thank you again.

---
*All for an existing project, which differs from step 1 of my reproduction above, so I thought I should mention it explicitly
